app/app/urls.py

```diff
@@ -71,6 +71,7 @@
     url('^api/v1/bounty/fulfill', dashboard.views.fulfill_bounty_v1, name='fulfill_bounty_v1'),
     path('api/v1/bounty/<int:bounty_id>/close', dashboard.views.close_bounty_v1, name='close_bounty_v1'),
     path('api/v1/bounty/payout/<int:fulfillment_id>', dashboard.views.payout_bounty_v1, name='payout_bounty_v1'),
+    path('api/v1/bounty/payout/broadcast/<str:tenant>', dashboard.views.payout_tx_forwarder_v1, name='payout_tx_forwarder_v1'),
```

Comment:
Thoughts on renaming it into something more generic `/api/v1/reverse-proxy/<str:tenant>`
as this might be used in grants as well

Reply:
nice one!